src/rqt_graph/rosgraph2_impl.py

```diff
@@ -30,9 +30,7 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 #
-# Revision $Id$
-
-from __future__ import print_function
+# Brawner: Adopted from ros_comm for ROS2
```

Comment:
It looks like this file was copied ROS 1 by directly committing to the branch. Please use a PR to do so (and force push remove the commit on the target branch) in order to make this discoverable as well as document which exact version of the upstream file was copied.

Reply:
New PR is here: https://github.com/ros-visualization/rqt_graph/pull/23
This branch has been rebased with that PR.